test/application_system_test_case.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "capybara/rails"
+require "capybara/minitest"
+require "capybara/cuprite"
+
+Ferrum::Browser.new(process_timeout: 5)
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
+  driven_by :cuprite, using: :chrome, screen_size: [1400, 1400]
+
+  def with_preview(preview_name)
+    component_uri = self.class.name.split("Integration").first.underscore
+
+    visit("/rails/view_components/#{component_uri}/#{preview_name}")
```

Comment:
we also have the stories routes available here! wdyt of using them? 
e.g: `/rails/stories/primer/avatar_stack_component/avatar_stack`

Reply:
Interesting. I could go either way. I think we'll want to define specific previews for specific integration tests, but I could also see us reusing the Storybook infrastructure. Ultimately they are similar concepts that should probably share more logic than they do in this PR. 

Perhaps we can land what we have here and look into this further at a later time?